app/lib/api/error.rb

```
@@ -0,0 +1,37 @@
+# lib/api/errors.rb
+module Api
+  class Error < StandardError
+    def initialize(message, status)
+      @message = message
+      @status = status
+    end
```

rubocop (1.69.1)

[warning] Call `super` to initialize state of the parent class.

(Lint/MissingSuper)